### DISCUSSION TITLE
Waypoints serve as junctions for Train routing.

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#655] [General]
Subject line doesn;t make much sense but what I would like to see is that you can put down a waypoints and join them.. or have them connected much like you do logistics. Why?

Simple example. 
Two line rail. Two waypoints, one for each direction. Currently, when creating a route I have to find and add direction in the route that indicates the correct direction.  Sometimes that is unnecessary and very tedious. Sometimes I wish there was a junction option.,
IF I could take a small set of waypoints that are close to each other and set them as connected, call it a junction, I could then tell the train to use the junction as a waypoint much like you would a multi platform train station without having to indicate which rail (direction) to use.... "Just pass through there."  So now I would have one Junction to represent multiple waypoints so creating and reading my itinerary will be easier overall. 

Obviously it would need to remain optional which were connected and also be configurable.. IE: Changeable after created..

**Comment by victorleblais** [comment]
You can create waypoints with multiple tracks, like stations, and you can specify which track the train has to use, like stations.

So I think this fits what you are describing in your example.

![image](https://user-images.githubusercontent.com/4365075/116634696-1425b400-a95d-11eb-98a5-d55c6df62cc3.png)

**Reply by JezzyBelle** [reply]
Ok... So I did not see that you could make a (multiple platform) linked waypoint.. That is exactly what I was looking for. So thank you for opening my eyes! That helps alot!

HOWEVER...  I would still like to be able to take a cluster of waypoints **not directly attached** or even facing the same direction and be able to connect them logically like you have with the logistics connectivity between buildings. If in range, you can connect or disconnect one or more. I may not include all routes. I could link several of those waypoints within range and call them a Junction. Now, I can generally just say to my train, "Go through that area. You pick the rail or the route from there". 
For example, let's say I have a  number of rails intersecting in the middle of my territory and not all are easily routed through one multiple platform waypoint since they head off in different directions. Some may even head off in different directions BUT still have an alternate way to get to to my destination. 
Before, I may have had to designate a few different waypoints in my route to navigate it more effectively mso I make sure I don't take any of the unpreferred routes. This way, I may only have to designate a junction since I have excluded the unpreferred route(s) there. 
Not sure I can explain it better. All I know is.. LOL.. It would be something that I could make good use of in certain places.
